compiler-rt/lib/builtins/int_types.h

```diff
@@ -139,7 +139,6 @@ typedef union {
   udwords u;
   double f;
 } double_bits;
-#endif
 
 typedef struct {
```

Comment:
Is this struct not needed outside of the no-fp case?

Reply:
No, `uqwords` is only used in the subsequent declarations of the `xf_bits` union (for 80 bit long doubles) and the `tf_bits` union (for 128 bit quads).  (This is not the case for the earlier `udwords`, btw)